src/main/java/se/sundsvall/supportmanagement/service/mapper/CommunicationMapper.java

```diff
@@ -179,9 +180,9 @@ private CommunicationAttachmentEntity toMessageAttachment(final String namespace
 			.withMunicipalityId(municipalityId)
 			.withId(UUID.randomUUID().toString())
 			.withName(attachment.getName())
+			.withFileSize(attachment.getBase64EncodedString().length())
```

Comment:
Shouldn't this be `byteArray.length` instead? Otherwise it's not the actual file size, but rather "the number of string characters representing the encoded file in base64-format"?